docs/user-guide/containers.md

```diff
@@ -94,3 +94,23 @@ The relationship between Docker's capabilities and [Linux capabilities](http://m
 | SETFCAP |  CAP_SETFCAP |
 | WAKE_ALARM |  CAP_WAKE_ALARM |
 | BLOCK_SUSPEND |  CAP_BLOCK_SUSPEND |
+
+Capabilities are added/dropped in the [`SecurityContext`](http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_securitycontext), e.g.:
```

Comment:
Passive voice. If the user is responsible, please rephrase to "You can add or drop capabilities by editing the [`SecurityContext`]."